index.js

```diff
@@ -7,18 +7,18 @@
  */
 function exportToJsonString(idbDatabase, cb) {
   const exportObject = {};
-  const size = new Set(idbDatabase.objectStoreNames).size;
+  let objectStoreNames = new Set(idbDatabase.objectStoreNames);
```

Comment:
Please could we stick with `const`? We can use separate variables for the Set and the Array to avoid confusion.

Reply:
I'm not sure I see the benefit? It would be less changes to just keep it as is, instead of requiring separate variables.